main.go

```diff
@@ -16,18 +16,24 @@ package main
 
 import (
 	"flag"
-	"log"
+	"github.com/sirupsen/logrus"
+	"os"
 	"time"
 
-	"github.com/wjkohnen/prometheus_fileage_exporter/exporter"
+	"github.com/jwkohnen/prometheus_fileage_exporter/exporter"
 )
 
 func main() {
-	s := exporter.NewDefaultServer(exporter.NewExporter(configure()))
+	// Prepare logging
+	log := logrus.New()
+	log.Out = os.Stdout
```

Comment:
Please use stderr. Logging should always use stderr, except for request logs if the program is not a cgi server.